test/namespaced_dependency/source/namespaced_dependency.cpp

```diff
@@ -1,3 +1,4 @@
+#include <namespaced_dependency/namespaced_dependency.h>
```

Comment:
Note: normal you include your interface header as first.
I you do, you are sure it compiles standalone.